airbyte-cdk/java/airbyte-cdk/core/src/main/kotlin/io/airbyte/cdk/integrations/destination/buffered_stream_consumer/BufferedStreamConsumer.kt

```diff
@@ -288,7 +288,8 @@ internal constructor(
                  * hasFailed=false, then it could be full success. if hasFailed=true, then going for partial
                  * success.
                  */
-                onClose.accept(false, null)
+                // TODO what to do here?
+                onClose.accept(false, HashMap())
```

Comment:
@gisripa is this class still in use? (I'd like to kill it if not...)

Reply:
some very old connectors and random reason bigQuery has an import for a stale javadoc 😂

Reply:
ugh. whatever, I'll add stream status tracking here too I guess. Probably easier than trying to track down all the indirect usages :sob: